### PR TITLE
copy fix and command

### PR DESCRIPTION
**Pull request by sileht**
## fix(copy): delete temporary branches

copy branches starts by mergify/copy. We only clean mergify/bp branches.

This change also cleanup copy branches

Fixes MRGFY-385

## feat: allow copy command

Fixes MRGFY-384

### CODE REVIEW

**Comment by sileht** [comment]
I have already set `MERGIFYENGINE_BOT_USER_LOGIN` in heroku

File: mergify_engine/data/default_pull_request_rules.yml [hunk 1]
```diff
@@ -1,9 +1 @@
-rules:
-  - name: delete backport branch (Mergify rule)
-    hidden: true
-    conditions:
-      - author=mergify[bot]
-      - head~=^mergify/bp/
-      - closed
-    actions:
-      delete_head_branch: {}
+
```
Comment (jd) [inline mergify_engine/data/default_pull_request_rules.yml:1]: if the file is empty we could probably just remove it

**Review by jd — CHANGES_REQUESTED** [review]

**Review by jd — APPROVED** [review]

**Comment by sileht** [comment]
@mergifyio update

**Comment by sileht** [comment]
@Mergifyio update

**Pull request merged**